SparkPost/2web2ui
# FE-783 - Adding additional Cypress tests to the billing page

[FE-783](https://jira.int.messagesystems.com/browse/FE-783)

### What Changed
- Updates default account fixture to higher level plan to show additional UI states
- Updates Cypress to `4.0.2`
- Fleshes out billing page Cypress integration tests
- updates `cy.assertLink` method to use `cy.findBy*` instead of `cy.queryBy*` to take advantage of baked in timeout

### How To Test
- Run tests locally via `npm run test-integration`
- Go through the billing page and verify page states are well tested by new changes

### To Do
